pynest/nest/tests/test_sp/test_growth_curves.py

```diff
@@ -289,7 +289,8 @@ def setUp(self):
 
         # build
         self.pop = nest.Create('iaf_psc_alpha', 10)
-        self.local_nodes = nest.GetNodes([0], {'model': 'iaf_psc_alpha'}, True)[0]
+        local_nodes = nest.GetNodes([0], {'model': 'iaf_psc_alpha'}, True)
+        self.local_nodes = local_nodes[0]
         self.spike_detector = nest.Create('spike_detector')
```

Comment:
Why is this in this pull request?

Reply:
Because PEP8 requested me to make the line shorter a dnI tried to do it in a nicer way than just breaking it.